pkg/osvscanner/osvscanner.go

```diff
@@ -684,6 +689,18 @@ func filterPackageVulns(r reporter.Reporter, pkgVulns models.PackageVulns, confi
 	return pkgVulns
 }
 
+// isUnimportant returns if a Debian vulnerability is unimportant
+// Details: https://security-team.debian.org/security_tracker.html#severity-levels
+func isUnimportant(affectedPackages []models.Affected) bool {
```

Comment:
nit: better for the Debian check to live here too. This way we keep all the related logic in one place.